server/sonar-server/src/main/java/org/sonar/server/computation/step/ParseReportStep.java

```diff
@@ -69,7 +77,7 @@ public void execute(ComputationContext context) {
 
       // and parse!
       int rootComponentRef = reportMetadata.getRootComponentRef();
-      recursivelyProcessComponent(reader, context, rootComponentRef);
+      recursivelyProcessComponent(reader, context, rootComponentRef, defaultAssignee(context.getProjectSettings().getString(DEFAULT_ISSUE_ASSIGNEE)));
```

Comment:
I mean I check in the database if the login exists. I don't want to assign it to an issue if the login doesn't exist in the database.